spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkSQLProperties.java

```diff
@@ -90,4 +90,8 @@ private SparkSQLProperties() {}
   public static final String EXECUTOR_CACHE_LOCALITY_ENABLED =
       "spark.sql.iceberg.executor-cache.locality.enabled";
   public static final boolean EXECUTOR_CACHE_LOCALITY_ENABLED_DEFAULT = false;
+
+  // Controls whether to calculate column statistics and report them to Spark
+  public static final String ENABLE_COLUMN_STATS = "spark.sql.iceberg.enable-column-stats";
+  public static final boolean ENABLE_COLUMN_STATS_DEFAULT = false;
```

Comment:
Why don't we want to default to true (other than behavior change?)

Reply:
Changed the default to `true`. Thanks